frontend/server/src/Controllers/Contest.php

```diff
@@ -1759,9 +1773,14 @@ private static function validateCommonCreateOrUpdate(
             'penalty_calc_policy',
             ['sum', 'max']
         );
+        \OmegaUp\Validators::validateOptionalStringNonEmpty(
+            $r['problems'],
+            'problems'
+        );
 
         // Problems is optional
         if (!is_null($r['problems'])) {
+            /** @var list<\OmegaUp\RequestProblem>|null */
             $requestProblems = json_decode($r['problems']);
             if (is_null($requestProblems)) {
```

Comment:
no es necesario la clase `RequestProblem`!

```suggestion
            /** @var list<array{problem: string, points: int}>|null */
            $requestProblems = json_decode($r['problems'], /*$assoc=*/true);
            if (!is_array($requestProblems)) {
```

Reply:
Listo, se quita la clase